.github/workflows/release.yaml

```diff
@@ -48,7 +48,9 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-receiver
+          images: |
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-receiver
```

Comment:
I think GHCR only allows images for the specific repo name, but not if they have such a suffix (which is also why the release CI step failed last time). Maybe we only push to Docker Hub? 

Or a single image that can be used for both sender and receiver?